js/src/data/reducer.js

```diff
@@ -524,6 +525,16 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 			return setIn( state, 'gtinMigrationStatus', data?.status );
 		}
 
+		case TYPES.UPDATE_ADS_ENHANCED_CONVERSION: {
+			const { status: enableEnhancedConversions } = action;
```

Comment:
@ankitrox Any idea why we need to rename `status` to `enableEnhancedConversions`?